recipe/meta.yaml

```diff
@@ -42,7 +42,7 @@ outputs:
         - cmake
         - make  # [not win]
       host:
-        - tiledb =2.18.2
+        - tiledb 2.18.*
```

Comment:
Recall that we purposefully built libtiledbsoma against libtiledb 2.18.2 so that the conda solver could be flexible about which libtiledb it installed when building the R and Python variants

https://github.com/TileDB-Inc/tiledbsoma-feedstock/pull/74#issuecomment-1885371268

If you switch back to `2.18.*`, this will build libtiledbsoma against 2.18.4, and force 2.18.4 to be installed for all the downstream builds (which appear to not solve)

Reply:
@jdblischak and cc @ihnorton we definitely need to use 2.18.4 either via `2.18.*` or via `2.18.4` ...

Reply:
OK thanks @jdblischak !! I hadn't grasped correctly; thanks for straightening me out :)